geopandas/io/tests/test_arrow.py

```diff
@@ -687,22 +687,30 @@ def test_write_spec_version(tmpdir, format, version):
 
 @pytest.mark.parametrize("version", ["0.1.0", "0.4.0"])
 def test_read_versioned_file(version):
-    # Verify that files for different metadata spec versions can be read
-    # created for each supported version:
-    # df = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
-    # df.head(2).to_feather(DATA_PATH / 'arrow' / f'naturalearth_lowres_top2_v{METADATA_VERSION}.feather')  # noqa: E501
-    # df.head(2).to_parquet(DATA_PATH / 'arrow' / f'naturalearth_lowres_top2_v{METADATA_VERSION}.parquet')  # noqa: E501
-
+    """
+    Verify that files for different metadata spec versions can be read
+    created for each supported version:
+
+    # small dummy test dataset (not naturalearth_lowres, as this can change over time)
+    from shapely.geometry import box, MultiPolygon
+    df = geopandas.GeoDataFrame(
+        {"col_str": ["a", "b"], "col_int": [1, 2], "col_float": [0.1, 0.2]},
+        geometry=[MultiPolygon([box(0, 0, 1, 1), box(2, 2, 3, 3)]), box(4, 4, 5,5)],
+        crs="EPSG:4326",
```

Comment:
@brendan-ward I changed this feather/parquet test to use a small dummy dataset instead of naturalearth (since that is not necessarily stable, and then the expected is no longer correct). 

I recreated the files for 0.1.0 with geopandas 0.10, and for 0.4.0 with the main branch.

Reply:
Thank you!  In hindsight, it was definitely a bad idea to create a point-in-time test file based on naturalearth, so a dummy dataset is _much_ better in this case.